fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/explain/NodeInfo.java

```diff
@@ -64,6 +64,13 @@ public class NodeInfo {
             NodeIcon.DATA_ACCESS_OPERATOR,
             "Covering Index Scan",
             "A covering index scan operator uses a secondary index to quickly find records in a given search range without looking up the base records.");
+
+    public static final NodeInfo AGGREGATE_INDEX_SCAN_OPERATOR = new NodeInfo(
```

Comment:
Can we just use the regular index `NodeInfo`? If you must use this one, please change the explanation as it does not fetch base records.

Reply:
I thought it is more explicit to use own `NodeInfo`, but I am ok with also using a regular index scan operator explanation since it is logically the same.